ui/src/components/DenylistTable/index.jsx

```diff
@@ -0,0 +1,158 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { string, shape, func, arrayOf } from 'prop-types';
+import { pipe, map, sort as rSort } from 'ramda';
+import memoize from 'fast-memoize';
+import { camelCase } from 'change-case/change-case';
+import { withStyles } from '@material-ui/core/styles';
+import TableRow from '@material-ui/core/TableRow';
+import TableCell from '@material-ui/core/TableCell';
+import Typography from '@material-ui/core/Typography';
+import { notificationAddress, pageInfo } from '../../utils/prop-types';
+import { VIEW_DENYLISTED_NOTIFICATIONS_PAGE_SIZE } from '../../utils/constants';
+import sort from '../../utils/sort';
+import ConnectionDataTable from '../ConnectionDataTable';
+
+const sorted = pipe(
+  rSort((a, b) => sort(a.node.notificationAddress, b.node.notificationAddress)),
+  map(({ node: { notificationAddress } }) => notificationAddress)
+);
+const tableHeaders = ['Notification Type', 'Notification Address'];
```

Comment:
The address can be an email address, pulse routing key or an IRC user or channel name. Maybe we can rename them to `Type` and `Address` since, as you said, the dashboard title already indicates the purpose of the table

Reply:
Good plan.